apps/vscode/src/main.ts

```diff
@@ -42,6 +45,10 @@ import {
   NxProjectTreeItem,
   NxProjectTreeProvider,
 } from '@nx-console/vscode/nx-project-view';
```

Comment:
I moved `verifyWorkspace` from from the `verify` lib to the `nx-workspace` lib to resolve a circular dependency between the two.